src/components/Modal/README.md

```diff
@@ -212,6 +226,17 @@ export default {
 };
 </script>
 ```
+### Configurable options
+The `modalApi.open()` function has a second optional object parameter that you allows for configurable options. Current available options are:
+
+```html
+{
+	// Modal will close when clicked outside of it - default false
+	closeOnClickOutside: boolean,
+	// function to run before modal is closed. If return value is false modal will not close.
+	beforeCloseHook: () => { return boolean },
```

Comment:
This hook should be on Modal since that's the file that would be concerned about the sensitivity of its contents. 

API should look like this
```vue
<o-modal :before-close="beforeCloseHook">
```